.github/workflows/mddiffcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Check CAP Diffs'
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  mddiffcheck:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
```

Comment:
do you need to setup go? I see from https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md that go should already be installed?

Reply:
The tools are cached, but not setup. The setup-go action will setup the tool for use, it won't download it unless the cache hasn't provided it. The setup-go action uses the tool-cache library to do that.